### PR TITLE
[GHA] black format everything automatically

### PR DESCRIPTION
**Pull request by dschwoerer**
Automatically apply black formatting to all python files

### CODE REVIEW

**Review by ZedThree — DISMISSED** [review]

File: .github/workflows/black-fix.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+    - name: Run black
+      run: |
+        pwd
+        ls
+        $HOME/.local/bin/black tests/ tools/ $(grep '^#.*python.*$' bin/ tests/ tools/ -Ir|cut -d: -f1|grep -v _boutcore_build)
+
+    - uses: stefanzweifel/git-auto-commit-action@v4
+      with:
+        commit_message: Apply black changes
```
Comment (ZedThree) [inline .github/workflows/black-fix.yml:37]: Does this trigger another CI run? If so, might be good to put `[skip CI]` in the message

**Comment by ZedThree** [comment]
Nice! Thanks @dschwoerer !

File: .github/workflows/black-fix.yml (hunk 1 above)
Comment (dschwoerer) [reply .github/workflows/black-fix.yml]: Yes it does, I added [skip CI] :+1:

**Review by ZedThree — DISMISSED** [review]

**Comment by dschwoerer** [comment]
`[skip CI]` doesn't seem to skip - should be `[skip ci]`
should have [rtfm](https://github.blog/changelog/2021-02-08-github-actions-skip-pull-request-and-push-workflows-with-skip-ci/)

**Review by ZedThree — APPROVED** [review]

**Pull request merged**